paymentsheet/api/paymentsheet.api

```diff
@@ -15,6 +15,7 @@ public abstract interface class com/stripe/android/customersheet/CustomerAdapter
 	public abstract fun retrieveSelectedPaymentOption (Lkotlin/coroutines/Continuation;)Ljava/lang/Object;
 	public abstract fun setSelectedPaymentOption (Lcom/stripe/android/customersheet/CustomerAdapter$PaymentOption;Lkotlin/coroutines/Continuation;)Ljava/lang/Object;
 	public abstract fun setupIntentClientSecretForCustomerAttach (Lkotlin/coroutines/Continuation;)Ljava/lang/Object;
+	public abstract fun updatePaymentMethod (Lcom/stripe/android/model/PaymentMethodUpdateParams;Lkotlin/coroutines/Continuation;)Ljava/lang/Object;
```

Comment:
Is `CustomerAdapter` intended to be public? If so, should we look into hiding the `updatePaymentMethod` method? We would need some `InternalCustomerAdapter` for just this use case.